ExposureNotification/ViewControllers/UploadKeysViewController.swift

```diff
@@ -415,6 +407,25 @@ extension UploadKeysViewController: UITextFieldDelegate {
     }
 }
 
+extension UploadKeysViewController: UIGestureRecognizerDelegate {
+    func gestureRecognizerShouldBegin(_ gestureRecognizer: UIGestureRecognizer) -> Bool {
+        
+        // The views that should not be responsed to the tap gesture.
```

Comment:
There is a violation of grammar.

This sentence should be
```
// The views that should not response to the tap gesture.
```

Reply:
Thanks.